lib/resources/majors.tsx

```diff
@@ -131,14 +131,460 @@ import { Major, Minor } from './commonTypes';
+        },
+        {
+          description:
+            '<b>Quantum Mechanics I</b> <br /> AS.171.303 Quantum Mechanics I',
+          required_credits: 4,
+          criteria: 'AS.171.303[C]',
+        },
+        {
+          description:
+            '<b>Quantum Mechanics II (or Topics in Modern Physics)</b> <br /> AS.171.304 Quantum Mechanics II',
+          required_credits: 4,
+          criteria: 'AS.171.304[C]',
+        },
+        {
+          description:
+            '<b>Advanced Physics Laboratory</b> <br /> AS.173.308 Advanced Physics Laboratory',
+          required_credits: 3,
+          criteria: 'AS.173.308[C]',
+        },
+      ],
+    },
+    {
+      name: 'Electives',
+      required_credits: 15,
+      min_credits_per_course: 3,
+      exclusive: true,
+      pathing: true,
+      description:
+        'Five elective courses must be taken at the 200-600 level. Four courses must be within a single department or program in KSAS or WSE: ' +
+        'Physics and Astronomy, Biology, Biophysics, Chemistry, Cognitive Science, Earth and Planetary Sciences, ' +
+        'Mathematics, Neuroscience and/or the School of Engineering (excluding courses listed as 500.xxx, 660.xxx, 551.xxx and 661.xxx). ' +
+        'These courses must constitute a coherent and rigorous program of study.',
+      criteria:
+        '(AS Physics & Astronomy[D]^OR^AS Biology[D]^OR^AS Biophysics[D]^OR^AS Chemistry[D]^OR^AS Cognitive Science[D]^OR^' +
+        'AS Earth & Planetary Sciences[D]^OR^AS Mathematics[D]^OR^AS Neuroscience[D])^AND^(200[L]^OR^Upper Level[L])', // TODO: add wse
+      fine_requirements: [
+        {
+          description: '<b>Physics and Astronomy</b>',
+          required_credits: 12,
+          criteria: 'AS Physics & Astronomy[D]',
+        },
+        {
+          description: '<b>Biology</b>',
+          required_credits: 12,
+          criteria: 'AS Biology[D]',
+        },
+        {
+          description: '<b>Biophysics</b>',
+          required_credits: 12,
+          criteria: 'AS Biophysics[D]',
+        },
+        {
+          description: '<b>Chemistry</b>',
+          required_credits: 12,
+          criteria: 'AS Chemistry[D]',
+        },
+        {
+          description: '<b>Cognitive Science</b>',
+          required_credits: 12,
+          criteria: 'AS Cognitive Science[D]',
+        },
+        {
+          description: '<b>Earth and Planetary Sciences</b>',
+          required_credits: 12,
+          criteria: 'AS Earth & Planetary Sciences[D]',
+        },
+        {
+          description: '<b>Mathematics</b>',
+          required_credits: 12,
+          criteria: 'AS Mathematics[D]',
+        },
+        {
+          description: '<b>Neuroscience</b>',
+          required_credits: 12,
+          criteria: 'AS Neuroscience[D]',
+        },
+        {
+          description: '<b>Engineering</b>',
+          required_credits: 12,
+          criteria: '', // TODO: add WSE
```

Comment:
criteria string for WSE? `200-600 level in  the School of Engineering (excluding courses listed as 500.xxx, 660.xxx, 551.xxx and 661.xxx)`